librz/asm/arch/rl78/rl78.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: 2023 Bastian Engel <bastian.engel00@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RL78_H
+#define RL78_H
+
+#include "instr.h"
+
+#include <rz_util.h>
+#include <rz_types.h>
+
+#include <stddef.h>
```

Comment:
These two are unnecessary

Reply:
rz_strf() doesn't seem to be usable for buffers that have already decayed to pointers